### ISSUE TITLE
Can't connect with old TLS version

### ISSUE THREAD

**Issue by e2002e** [sqlmapproject/sqlmap#5724]
I can't connect to a server that is running an old tls version.
```
[11:39:10] [INFO] searching for links with depth 1
[11:39:10] [DEBUG] SSL connection error occurred for 'PROTOCOL_TLS_CLIENT' ('SSLError: [SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:992)')
[11:39:10] [DEBUG] SSL connection error occurred for 'PROTOCOL_TLSv1_2' ('SSLError: [SSL: WRONG_SSL_VERSION] wrong ssl version (_ssl.c:992)')
[11:39:10] [DEBUG] SSL connection error occurred for 'PROTOCOL_TLSv1_1' ('SSLError: [SSL] internal error (_ssl.c:992)')
[11:39:10] [DEBUG] SSL connection error occurred for 'PROTOCOL_TLSv1' ('SSLError: [SSL] internal error (_ssl.c:992)')
```

 - sqlmap 1.8.5.4#dev
 - Installation method: github
 - Operating system: Debian 12
 - Python version 3.11

**Comment by stamparm** [comment]
using DEBUG in sqlmap for "debugging" TLS/SSL connections is of no help here

you should inspect on what TLS/SSL versions is your web site supporting in the first place, and report back here. I am pretty sure that this is not a sqlmap's fault

**Comment by stamparm** [comment]
https://stackoverflow.com/a/40557750

**Comment by e2002e** [comment]
I couldn't find out the ssl version.
I was on debian and using the browser, i could connect to the website only after enabling deprecated tls. But none of the programs I used could connect. The website's certificate is self signed. I tried downloading the .pem file and pass --cacert to curl but still unsupported ssl version error. I also had changed openssl.cnf.

Using sslscan tells me that actualy none of the ssl/tls protocols are supported.
Now i'm running kali and I don't got the same errors, even the browser can't connect.

So yes that isn't an issue with sqlmap.
